mail-client/neomutt/neomutt-20200626-r1.ebuild

```diff
@@ -94,11 +94,20 @@ src_configure() {
 		"--sysconfdir=${EPREFIX}/etc/${PN}"
 		"$(use_enable ssl)"
 		"$(use_enable gnutls)"
+
+		"$(usex test --testing)"
 	)
 
 	econf CCACHE=none "${myconf[@]}"
 }
 
+src_test() {
+	local test_dir=$(mktemp --directory)
```

Comment:
I think we may need to either put || die inside the [subshell](https://devmanual.gentoo.org/ebuild-writing/error-handling/index.html) (ok on >= EAPI 7) or check if it's a directory with a Bash test ([[ -d ... ]]) afterwards, and die if not.